### PR TITLE
TPD-1614 Adding db function SplitIntsKey to order by the Key instead of peopleid.

### PR DESCRIPTION
**Pull request by josearmandohf**
[touchpointsoftware.atlassian.net/browse/TPD-1614](https://touchpointsoftware.atlassian.net/browse/TPD-1614)

### CODE REVIEW

File: CmsData/CmsData.csproj [hunk 1]
```diff
@@ -1247,6 +1248,7 @@
     <Resource Include="Classes\OnlineRegSummaryText\Details2.xml">
       <SubType>Designer</SubType>
     </Resource>
+    <Content Include="Migrations\20201112_create_SplitIntsKey_function.sql" />
```
Comment (brondavies) [inline CmsData/CmsData.csproj:1251]: Reminder: Don't include migrations in the project file

File: CmsWeb/Areas/Giving/Models/GivingPage/GivingPageModel.cs [hunk 2]
```diff
@@ -71,8 +71,9 @@ public List<GivingPageItem> GetGivingPages(int id = 0)
                                               }).SingleOrDefault(),
                                 TopText = gp.TopText,
                                 ThankYouText = gp.ThankYouText,
-                                OnlineNotifyPerson = (from np in db.People
-                                                      where onp.Contains(np.PeopleId)
+                                OnlineNotifyPerson = (from p in onp
+                                                      join np in db.People on p.ValueX equals np.PeopleId
+                                                      orderby p.TokenID
```
Comment (brondavies) [inline CmsWeb/Areas/Giving/Models/GivingPage/GivingPageModel.cs:76]: This should fix the problem of ordering without the new function that is essentially the same as SplitInts

**Review by brondavies — CHANGES_REQUESTED** [review]
I don't think we need the new SplitIntsKey function

File: CmsData/CmsData.csproj (hunk 1 above)
Comment (josearmandohf) [reply CmsData/CmsData.csproj:1251]: Right, removing it...

**Review by brondavies — APPROVED** [review]

**Pull request merged**